docs/ReleaseProcess.md

```diff
@@ -25,41 +27,69 @@ eBPF for Windows, and how to service it later.
+13. Publish the release as per the [Publishing a Release](ReleaseProcess.md#publishing-a-release) process.
+
+## Release Branch Validation
+
+The `CI/CD - Release validation` worfklow (`cicd-release-validation.yml`) is used to validate a release branch. It contains more tests than the regular CI/CD pipeline, including longer duration fuzz tests, fault injection tests, stress tests, performance tests etc. These tests can be manually scheduled. Release manager must run these tests on the release manager. Due to the non-deterministic nature of some of the tests, it is recommended that the tests are run at least three times on the branch. If any of the tests fail, the release manager must investigate the failure and follow up with issues in github. Once potential fixes are merged to the release branch repeat the process until the workflow completes successfully.
+
+## Publishing the Release to GitHub
+
+6. Go to the repo on GitHub and click on "`<Code>`" and click on right the "`Create a new release`" link.
+7. Click on the "`Choose a tag`" combo box and select the tag with version number for the release, as created earlier.
+8. Fill in the release title as "`vX.Y.Z`". Note "`Z`" must be `0` for the monthly release. Otherwise, it should be the patch number.
+9. Manually enter release notes or click "`Generate release notes`" and then edit as desired.
+10. Attach the `.msi`, the (non-redist) `.nupkg`, the `Build-x64-[Release|Debug].zip` and the `Build-x64-native-only-[Release|Debug].X.Y.Z.zip` build (from the CI/CD artifacts), by dropping them in the "`Attach binaries by dropping them here or selecting them.`" area. For example, the file list for `v0.11.0` should be:
+
+    - *Build-x64-native-only-Debug.0.11.0.zip*
+    - *Build-x64-native-only-Release.0.11.0.zip*
+    - *ebpf-for-windows.0.11.0.msi*
+    - *eBPF-for-Windows.0.11.0.nupkg*
+    - *Build-x64-Debug.zip*
+    - *Build-x64-Release.zip*
+
+11. Check the "`Set as a pre-release`" checkbox, unless the release is production-signed.
+12. Once the uploads are complete, click "`Publish release`".
+
+## Publishing the Release to NuGet.org
 
-    - `eBPF-for-Windows.X.Y.Z.nupkg`
-1. Submit the PR for review (from its draft state), and wait for it to be approved and merged into the main repo's "`main`" branch.
-1. On the main `ebpf-for-windows` repo, create a new release branch from `main` **corresponding to the previous PR's commit**, name it "`release/X.Y`" (replace "X.Y" with the version number being released).
-1. Publish the release as per the "[Publishing a Release](ReleaseProcess.md#publishing-a-release)" process.
+Upload the (**non-redist**) `.nupkg` file to [NuGet.org](https://www.nuget.org/) (the metadata inside the `.nuget` package itself will automatically populate all the other form fields).
```

Comment:
[Your nuget](https://www.nuget.org/packages/eBPF-for-Windows) is not in the microsoft account? Nor a specific one for eBPF? FYI, see the [msquic one](https://www.nuget.org/packages/Microsoft.Native.Quic.MsQuic.OpenSSL).

Reply:
@gtrevi pleae follow up with @nibanks .